core/imageroot/var/lib/nethserver/node/actions/get-facts/50get

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import json
+import sys
+
+platform = {}
+with open('/etc/os-release', encoding='utf-8') as file:
+    for line in file:
+        key, value = line.rstrip().split('=')
+        platform[key] = value.strip('"')
+
+facts = {
+    'distro': {
+        'name': platform['ID'],
+        'version': platform['VERSION_ID']
+    },
+    # TODO: Replace when version is included in release
+    'version': '8.0.0'
```

Comment:
What about reading the core image version instead?

    /etc/nethserver/core.env

It can be a tag like 1.2.3, or a development branch name

Reply:
Sure, however, by using `core_url='ghcr.io/nethserver/core:ns8-stable` in the [install script](https://github.com/NethServer/ns8-core/blob/9a75d2008880a0735466aa489be14d7c17fb6453/core/install.sh#L28) this information is lost. Shall we update the install.sh script every time? Otherwise we can update the image with a label and then during the installation save it elsewhere.